### PR TITLE
Fix/jr2 661

### PR DESCRIPTION
**Pull request by tdascoli**
Fix known mobile issues

### CODE REVIEW

File: src/main/webapp/app/candidate-search/candidate-detail/candidate-detail.component.html [hunk 1]
```diff
@@ -49,8 +49,8 @@ <h2 class="text--light">{{ relevantJobExperience.occupation }}</h2>
             </div>
 
             <!-- Occupation -->
-            <div *ngIf="jobExperiences$ | async as jobExperiences" class="candidate-detail__content">
-                <h4 class="candidate-detail__content__title candidate-detail__content__title--occupation">{{ 'candidate-detail.job-experience.title' | translate }}</h4>
+            <div *ngIf="candidateProfile.jobExperiences.length" class="candidate-detail__content">
```
Comment (birom) [inline src/main/webapp/app/candidate-search/candidate-detail/candidate-detail.component.html]: I think the ngIf should be the following:
```
*ngIf="jobExperiences$ | async as jobExperiences"
```
This was changed here: https://github.com/alv-ch/jobroom2/commit/1ff7738c94c482f4d079d3cf0fa42077fe236c68

File: src/main/webapp/app/candidate-search/candidate-search-filter/candidate-search-filter.component.ts [hunk 2]
```diff
@@ -32,6 +32,7 @@ import {
     styleUrls: ['./candidate-search-filter.component.scss']
 })
 export class CandidateSearchFilterComponent implements OnInit, OnDestroy {
+    isFilterCollapsed: boolean;
```
Comment (birom) [inline src/main/webapp/app/candidate-search/candidate-search-filter/candidate-search-filter.component.ts]: Please add a todo to review if the isFilterCollapsed should be part of the CandidateSearchState or not.

File: src/main/webapp/app/job-search/job-search-sidebar/job-search-filter/job-search-filter.component.ts [hunk 3]
```diff
@@ -20,6 +20,7 @@ export const COMPANY_DEBOUNCE_TIME = 500;
     styleUrls: ['./job-search-filter.component.scss']
 })
 export class JobSearchFilterComponent implements OnInit, OnDestroy {
+    isFilterCollapsed: boolean;
```
Comment (birom) [inline src/main/webapp/app/job-search/job-search-sidebar/job-search-filter/job-search-filter.component.ts]: Please add a todo to review if the isFilterCollapsed should be part of the JobSearchState or not.

File: src/main/webapp/app/candidate-search/candidate-search-filter/candidate-search-filter.component.ts (hunk 2 above)
Comment (tdascoli) [reply src/main/webapp/app/candidate-search/candidate-search-filter/candidate-search-filter.component.ts]: dove

File: src/main/webapp/app/job-search/job-search-sidebar/job-search-filter/job-search-filter.component.ts (hunk 3 above)
Comment (tdascoli) [reply src/main/webapp/app/job-search/job-search-sidebar/job-search-filter/job-search-filter.component.ts]: done

**Pull request merged**